package.json

```diff
@@ -1,35 +1,24 @@
 {
-  "name": "e-commerce-ts",
+  "name": "e-commerce-ones-and-zeroes-bn",
   "version": "1.0.0",
-  "description": "",
+  "description": "This is backend part of e-commerce-ones-and-zeroes",
   "main": "server.js",
-  "type": "module",
-  "scripts": {
-    "coverage": "nyc report --exclude=src/server.js --reporter=text-lcov | coveralls",
+  "scripts": { 
     "build": "tsc",
-    "test": "mocha --require @babel/register",
-    "start": "nodemon dist/server.js"
+    "start": "node dist/server.js",
+    "dev": "nodemon --exec ts-node src/server.ts"
   },
-  "keywords": [],
   "author": "",
   "license": "ISC",
-  "devDependencies": {
-    "@babel/core": "^7.24.4",
-    "@babel/node": "^7.23.9",
-    "@babel/preset-env": "^7.24.4",
-    "@babel/preset-typescript": "^7.24.1",
-    "@babel/register": "^7.23.7",
-    "@types/express": "^4.17.21",
-    "express": "^4.19.2",
-    "nodemon": "^3.1.0",
-    "ts-node": "^10.9.2",
-    "typescript": "^5.4.3"
```

Comment:
We need to install Jest for our testing

Reply:
At the moment, we don't need jest. It will be installed by the one who will setup tests. Until know we have not decided which tests we will use.